osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSliderRepeat.cs

```diff
@@ -18,17 +23,84 @@ public DrawableSliderRepeat(Beat hitObject)
         {
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            Alpha = 0.0f;
+            AlwaysPresent = true;
+        }
+
+        public Drawable InnerDrawableBox;
+
         protected override void LoadComplete()
         {
             base.LoadComplete();
-            DrawableBox.Size = Vector2.Multiply(DrawableBox.Size, 0.75f);
+            AddInternal(InnerDrawableBox = new Container
```

Comment:
Why aren't we overriding `DrawableBox` here?

Reply:
We need both the box and another inner box for rendering. The slider path uses their draw quads to render the mask and the outline.